sqllineage/core/handlers/source.py

```diff
@@ -72,6 +72,9 @@ def _handle_table(self, token: Token, holder: SubQueryLineageHolder) -> None:
             else:
                 # SELECT * FROM (tab2), which is valid syntax
                 self._handle(token.tokens[1], holder)
+        elif token.normalized.startswith("VALUES"):
```

Comment:
Once we get the token as VALUE, add another safeguard to make sure token.is_keyword is True. Also, let's make the existing expression token.normalized == "VALUES"

Reply:
Thank you! That works great. :)